compose/environment-variables/envvars-precedence.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Environment variables precedence
+description: Scenario Overview illustrating how environment variables are resolved in Compose
+keywords: compose, environment, env file
+redirect_from:
+- /compose/envvars-precedence/
+---
+
+When you set the same environment variable in multiple files, there’s a precedence rule used by Compose. It aims to resolve the value for the variable in question.
```

Comment:
As env variables could be defined in multiple ways and not only from files, it should be better to user sources or a synonym, wdyt?
```suggestion
When you set the same environment variable from multiple sources, there’s a precedence rule used by Compose. It aims to resolve the value for the variable in question.
```

Reply:
Oo yes, good point. Sources it is!